docs/content/guides/navigation/keyboard-shortcuts.md

```diff
@@ -0,0 +1,279 @@
+---
+id: 5w6juytv
+title: Keyboard shortcuts
+metaTitle: Keyboard shortcuts - JavaScript Data Grid | Handsontable
+description: Access all Handsontable features using just your keyboard. Use shortcuts you know from Google Sheets or Microsoft Excel.
+permalink: /keyboard-shortcuts
+canonicalUrl: /keyboard-shortcuts
+tags:
+  - key bindings
+  - keymap
+  - key mapping
+  - keyboard navigation
+  - hotkey
+  - accessibility
+  - function key
+  - commands
+react:
+  id: ddjw4zt8
+  metaTitle: Keyboard shortcuts - React Data Grid | Handsontable
+searchCategory: Guides
+---
+
+# Keyboard shortcuts
+
+Access all Handsontable features using just your keyboard. Use shortcuts you know from Google Sheets or Microsoft Excel.
```

Comment:
Hello, I propose the following version for consideration:

"All Handsontable features can be accessed by using the user's keyboard. Please refer to the Google Sheets or Microsoft Excel for the appropriate shortcuts."